Gordon360/Services/HousingService.cs

```diff
@@ -34,36 +34,73 @@ public IEnumerable<HousingViewModel> GetAll()
 
         /// <summary>
         /// Saves student housing info
+        /// - first, it creates a new row in the applications table and puts the id of the primary applicant and a timestamp
+        /// - second, it looks for the application id of the application with the information we just input (because 
+        /// the database creates the application ID so we have to ask it which number it generated for it)
+        /// - third, it inserts each applicant into the applicnts table along with the apartment ID so we know
+        /// which application on which they are an applicant
+        ///  
         /// </summary>
-        /// <returns>The housing item</returns>
-        public IEnumerable<HousingAppToSubmitViewModel>SaveApplication(string username, string sess_cde, string [] appIds)
+        /// <returns>Whether or not all the queries succeeded</returns>
+        public bool SaveApplication(string modifierId, string sess_cde, string [] appIds)
```

Comment:
Maybe it's good to indicate in the comment section that the modifierId is referring to the primary applicant.

Reply:
Yeah, and I could describe the other parameters while I am at it

Reply:
Can we explicitly specify in the comment that the modifierId is the same as the primary applicant ID.

Reply:
I think we should choose one term, (such as primary, modifier, editor, etc.) based on which one makes the most sense in the code and update the code so it all matches

Reply:
I would suggest editor, since it seems the best to communicate it's actual meaning to both students users and to developers

Reply:
We changed this in the code but still should change the database to editor. That would require a edmx remake, so we are
Waiting until we sort everything else in this branch out first.

Reply:
We made a note to fix this later because a lot is waiting on this PR.